test/gracenote/search_test.rb

```diff
@@ -2,12 +2,127 @@
 
 module Gracenote
   describe "#search" do
-    describe "with mode" do
+    let(:response_xml) do
+      <<-XML
+<RESPONSES>
+<RESPONSE STATUS="OK">
+  <RANGE>
+    <COUNT>10228</COUNT>
+    <START>1</START>
+    <END>10</END>
+  </RANGE>
+  <ALBUM ORD="1">
+    <GN_ID>7286646-4E6F2826F10FDDC14E358050AB130032</GN_ID>
+    <ARTIST>The Beatles</ARTIST>
+    <TITLE>Abbey Road</TITLE>
+    <PKG_LANG>ENG</PKG_LANG>
+    <DATE>1969</DATE>
+    <GENRE NUM="61364" ID="25332">60&apos;s Rock</GENRE>
+    <TRACK_COUNT>17</TRACK_COUNT>
+    <TRACK>
+      <TRACK_NUM>1</TRACK_NUM>
+      <GN_ID>7286647-0D4F7CB2CE2C57A95398CA34EA411CBD</GN_ID>
+      <TITLE>Come Together</TITLE>
+    </TRACK>
+  </ALBUM>
+  <ALBUM ORD="2">
+    <GN_ID>7317164-1E71D0380B7C84344423FABC8981BC1E</GN_ID>
+    <ARTIST>The Beatles</ARTIST>
+    <TITLE>Sgt. Pepper&apos;s Lonely Hearts Club Band</TITLE>
+    <PKG_LANG>ENG</PKG_LANG>
+    <DATE>1967</DATE>
+    <GENRE NUM="61364" ID="25332">60&apos;s Rock</GENRE>
+    <TRACK_COUNT>13</TRACK_COUNT>
+    <TRACK>
+      <TRACK_NUM>1</TRACK_NUM>
+      <GN_ID>7317165-9AD8800BA4FDB043F7AE8071692CB4BD</GN_ID>
+      <TITLE>Sgt. Pepper&apos;s Lonely Hearts Club Band</TITLE>
+    </TRACK>
+    <TRACK>
+      <TRACK_NUM>2</TRACK_NUM>
+      <GN_ID>7317166-79D0F85136FE2170885F97716EB779F9</GN_ID>
+      <TITLE>With A Little Help From My Friends</TITLE>
+    </TRACK>
+  </ALBUM>
+</RESPONSE>
+</RESPONSES>
+      XML
+    end
+
+    describe "#request" do
+      before do
+        stub_request(:post, //).to_return(headers: { 'Content-Type' => 'applicaiton/xml' }, body: response_xml)
```

Comment:
__[rubocop]__
* Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)